comment_scan.rb

```diff
@@ -85,16 +85,34 @@ def restart(num_to_post, bundle)
             end
           when 'huh?'
             matched_regexes = report_raw(comment["post_type"], comment["body_markdown"])
-            if matched_regexes.empty?
-              cb.say "Comment didn't match any regexes", room_id
+            reason_text = ""
+            #Go through regexes we matched to build reason_text
+            matched_regexes.each { |regex_matched| reason_text += "Matched reason \"#{Reason.where(id: regex_matched["reason_id"])[0]["name"]}\" for regex: #{regex_matched["regex"]}\n" }
+            #If post isn't deleted, check if this was an inactive comment
+            if !isPostDeleted(cli, comment["post_id"])
```

Comment:
In ruby, convention would be to name this method `post_deleted?` rather than `isPostDeleted` (yes, question marks are valid chars in method names in ruby)